examples/solid/test/Hello.test.jsx

```diff
@@ -0,0 +1,27 @@
+import { describe, expect, test } from 'vitest'
+import { fireEvent, render } from 'solid-testing-library'
+import { Hello } from '../components/Hello'
+
+describe('<Hello />', () => {
+  test('renders', () => {
+    const { container, unmount } = render(() => <Hello count={4} />)
+    expect(container.innerHTML).toMatchSnapshot()
+    unmount()
+  })
+
+  test('updates', async() => {
+    const { container, unmount, queryByText } = render(() => <Hello count={4} />)
+    const button = queryByText('x1')
+    const buttonClicked = new Promise((resolve) => {
```

Comment:
you forgot to await it
i would suggest using expect.anyAsserions() to check if expect is actually called

Reply:
I do await it on line 23. This is intentional.